app/Commands/PrsCommand.php

```
@@ -307,13 +314,14 @@ private function displayInteractive(array $prs): int
             '🔀 Check Merge Status',
             '💬 Manage Reviews',
             '🌐 Open in Browser',
-            '🔙 Back'
+            '🔙 Back',
         ];
-        $action = $this->choice(
-            'What would you like to do?',
-            $actions,
-            count($actions) - 1
+        $action = select(
```

PHPMD (2.15.0)

Avoid unused local variables such as '$action'. (Unused Code Rules)

(UnusedLocalVariable)